sklearn/preprocessing/data.py

```diff
@@ -1451,7 +1451,7 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
     >>> enc = OneHotEncoder()
     >>> enc.fit([[0, 0, 3], [1, 1, 0], [0, 2, 1], \
 [1, 0, 2]])  # doctest: +ELLIPSIS
-    OneHotEncoder(categorical_features='all', dtype=<... 'float'>,
+    OneHotEncoder(categorical_features='all', dtype=<... 'numpy.float64'>,
```

Comment:
wait so the type is the same but the string representation is different? that is somewhat confusing, isn't it?